src/main/groovy/com/github/spotbugs/snom/internal/SpotBugsRunnerForWorker.java

```diff
@@ -80,7 +83,16 @@ public void execute() {
 
       try {
         edu.umd.cs.findbugs.Version.printVersion(false);
-        edu.umd.cs.findbugs.FindBugs2.main(args);
+        try (FindBugs2 findBugs2 = new FindBugs2()) {
```

Comment:
Looks great. I'm hoping that this style of initializing a new `FindBugs2` instance for each worker execution should work well.

Reply:
hope that it works with the support from isolated process and classloader :)